lib/Doctrine/DBAL/Driver/Connection.php

```diff
@@ -99,7 +99,7 @@ public function rollBack();
     /**
      * Returns the error code associated with the last operation on the database handle.
      *
-     * @return string|null The error code, or null if no operation has been run on the database handle.
+     * @return string|bool|null The error code, or null if no operation has been run on the database handle.
```

Comment:
`bool|null` should be eliminated, not document.

Reply:
ignored